codegen/auth.codegen.ts

```diff
@@ -0,0 +1,27 @@
+import type { TypeScriptDocumentsPluginConfig } from '@graphql-codegen/typescript-operations';
```

Comment:
What do you think about using the graph prefix `zonos-auth-graph` like in `documentation`?

Reply:
Sounds good. Thanks for the suggestion.